src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,94 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFOULT_SIZE = 10;
+    private Pairs[] pairs;
+    private Pairs[] massiv;
+    private int index = 0;
+
+    public StorageImpl() {
+        pairs = new Pairs[DEFOULT_SIZE];
+        massiv = new Pairs[index];
+    }
+
     @Override
     public void put(K key, V value) {
+        Pairs pair = findPair(key);
```

Comment:
```suggestion
        Pair<K, V> pair = findPair(key);
```
dont use raw types))